src/fondant/compiler.py

```diff
@@ -143,17 +143,16 @@ def _generate_spec(
             safe_component_name = self._safe_component_name(component_name)
 
             component_op = component["fondant_component_op"]
-
-            # add metadata argument to command
-            command = ["--metadata", json.dumps(asdict(metadata))]
-
-            # add in and out manifest paths to command
-            command.extend(
-                [
-                    "--output_manifest_path",
-                    f"{path}/{safe_component_name}/manifest.json",
-                ],
-            )
+            execute_component = pipeline.execute_component(component_op)
+
+            command = [
+                "--metadata",
+                json.dumps(asdict(metadata)),
+                "--output_manifest_path",
+                f"{path}/{safe_component_name}/manifest.json",
+                "--execute_component",
```

Comment:
do we need this is the local compiler/runner ?

Reply:
It's gonna default for False now since caching is going to be disabled by default, but kept it for now for when we implement it in the future